drivers/can/mcp2515.c

```diff
@@ -0,0 +1,621 @@
+/*
+ * Copyright (c) 2018 Karsten Koenig
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <spi.h>
+#include <gpio.h>
+
+#define LOG_LEVEL CONFIG_CAN_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(mcp2515_can);
+
+#include "mcp2515.h"
+
+static int mcp2515_cmd_soft_reset(struct device *dev)
+{
+	u8_t cmd_buf[] = { MCP2515_OPCODE_RESET };
+
+	const struct spi_buf tx_buf = {
+		.buf = cmd_buf, .len = sizeof(cmd_buf),
+	};
+	const struct spi_buf_set tx = {
+		.buffers = &tx_buf, .count = 1U
+	};
+
+	return spi_write(DEV_DATA(dev)->spi, &DEV_DATA(dev)->spi_cfg, &tx);
+}
+
+static int mcp2515_cmd_bit_modify(struct device *dev, u8_t reg_addr, u8_t mask,
+									u8_t data)
```

Comment:
Looks like your editor uses 4 spaces tabs. Zephyr and Linux use 8 spaces tabs.

Reply:
I ran uncrustify again and it seems to have realigned them properly to 4 spaces tab

Reply:
Arrr 8 spaces tab I mean :-)